adcc/ExcitedStates.py

```diff
@@ -586,3 +654,60 @@ def describe_amplitudes(self, tolerance=0.01, index_format=None):
                 ret += "\n"
                 ret += separator
         return ret[:-1]
+
+    @property
+    def excitations(self):
+        """
+        Provides a list of Excitations, i.e., a view to all individual
+        excited states and their properties. Still under heavy development.
+        """
+        class Excitation:
+            pass
```

Comment:
Define this as a proper class with docs before the ExcitedStates. It is too important to be hidden in there. Actually it's constructor could take care of parsing "self" (just the ExcitedStates instance as an argument `parent` to the constructor). This would also structure this code better. Perhaps even move it to a different file to stress the importance of this class.

Reply:
Done.